sqlitecluster/SQLiteNode.cpp

```diff
@@ -1760,6 +1760,12 @@ void SQLiteNode::_onDisconnect(Peer* peer) {
 void SQLiteNode::_sendToPeer(Peer* peer, const SData& message) {
     SASSERT(peer);
     SASSERT(!message.empty());
+
+    // If a peer is currently disconnected, we can't send it a message.
+    if (!peer->s) {
+        PWARN("Can't send message to peer, no socket. Message '" << message.methodLine << "' will be discarded.");
```

Comment:
What's the difference between `PWARN` and `SWARN`?

Also, it might be interesting to know what peers we frequently can't send messages to, if that was something easy to add to this log line. If not, forget it.